code/modules/hydroponics/gene_modder.dm

```diff
@@ -391,6 +398,19 @@
 	else if(href_list["abort"])
 		operation = ""
 		target = null
+	else if(href_list["set_v"])
+		if(!seed)
+			return
+		var/V = sanitize(stripped_input(usr, "Choose variant name:", "Plant Variant Naming", seed.variant, MAX_MESSAGE_LEN))
```

Comment:
Might want to use a max length lower than 1024, or you could end up with absurdly long names that fill up the chat. Perhaps 64 characters is saner.

Reply:
Thanks, I copy pasted that from another file and didn't know what the max len did